lib/src/widgets/input_widget.dart

```diff
@@ -376,19 +377,12 @@ class _Item extends StatelessWidget {
   Widget build(BuildContext context) {
     return Container(
       child: Row(
+        textDirection: TextDirection.ltr,
         mainAxisAlignment: MainAxisAlignment.start,
         children: <Widget>[
-          country?.flagUri != null
-              ? Image.asset(
-                  country?.flagUri,
-                  width: 32.0,
-                  package: 'intl_phone_number_input',
-                )
-              : SizedBox.shrink(),
+          country?.flagUri != null ? Image.asset(country?.flagUri, width: 32.0) : SizedBox.shrink(),
```

Comment:
> ════════ Exception caught by image resource service ════════════════════════════════════════════════
The following assertion was thrown resolving an image codec:
Unable to load asset: assets/flags/af.png

When the exception was thrown, this was the stack: 
#0      PlatformAssetBundle.load (package:flutter/src/services/asset_bundle.dart:221:7)
<asynchronous suspension>
#1      AssetBundleImageProvider._loadAsync (package:flutter/src/painting/image_provider.dart:484:44)
#2      AssetBundleImageProvider.load (package:flutter/src/painting/image_provider.dart:469:14)
#3      ImageProvider.resolve.<anonymous closure>.<anonymous closure>.<anonymous closure> (package:flutter/src/painting/image_provider.dart:327:17)
...
Image provider: AssetImage(bundle: null, name: "assets/flags/af.png")
Image key: AssetBundleImageKey(bundle: PlatformAssetBundle#876af(), name: "assets/flags/af.png", scale: 1.0)
════════════════════════════════════════════════════════════════════════════════════════════════════

An exception will be thrown here because you missed out the package.

```suggestion
          country?.flagUri != null ? Image.asset(country?.flagUri, width: 32.0, package: 'intl_phone_number_input') : SizedBox.shrink(),
```